### DISCUSSION TITLE
Public Gitea host

### DISCUSSION THREAD

**Discussion by juristi** [magit/forge#434] [Ideas]
@tarsius : Is this mention in the documentation still valid? 

> Currently no public Gitea Hosts are known to me.

Just thought that have you checked out [Codeberg](https://codeberg.org) if their service would be good for the full Gitea support purposes?

**Comment by tarsius** [comment]
I'v updated that. Thanks for the report!

**Comment by tarsius** [comment]
I still don't plan to add support for this, or another, forge any time soon. "Add support for more forges", is still a long time goal, but until I get there, there is no need to keep an issue/discussion per forge open.
